### DISCUSSION TITLE
HxInputDate language

### DISCUSSION THREAD

**Discussion by mrjohnr** [havit/Havit.Blazor#443] [Q&A]
Hi
It is possible to change language for HxInputDate?

thanks

**Comment by hakenr** [comment]
Hi,
the language is taken from the current .NET thread language.

You can change it e.g. by using:
```csharp
CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
CultureInfo.DefaultThreadCurrentUICulture = new CultureInfo("en-US");
```

For more details see https://learn.microsoft.com/en-us/aspnet/core/blazor/globalization-localization
